pkg/kubefedctl/enable/validation.go

```diff
@@ -115,27 +116,7 @@ func federatedTypeValidationSchema(templateSchema map[string]v1beta1.JSONSchemaP
 												Type: "string",
 											},
 											"value": {
-												// Supporting the override of an arbitrary field
-												// precludes up-front validation.  Errors in
-												// the definition of override values will need to
-												// be caught during propagation.
-												AnyOf: []v1beta1.JSONSchemaProps{
-													{
-														Type: "string",
-													},
-													{
-														Type: "integer",
-													},
-													{
-														Type: "boolean",
-													},
-													{
-														Type: "object",
-													},
-													{
-														Type: "array",
-													},
-												},
+												XPreserveUnknownFields: pointer.BoolPtr(true),
```

Comment:
Do we need:
```suggestion
												XPreserveUnknownFields: pointer.BoolPtr(true),
												Type:                   "object",
```

Reply:
Honestly, I don't know. It works without it, too. 😬